bootstrap/roles/ocp4-install-noobaa/tasks/noobaa-create.yaml

```diff
@@ -0,0 +1,125 @@
+- name: Get cluster version
+  k8s_info:
+    api_version: config.openshift.io/v1
+    kind: ClusterVersion
+    name: version
+  register: r_cluster_version
+
+- name: Set ocp4_cluster_version fact
+  set_fact:
+    ocp4_cluster_version: "{{ r_cluster_version.resources[0].status.history[0].version }}"
+
+- name: Obtain Channel from Version
+  set_fact:
+    ocp4_channel: "{{ ocp4_cluster_version.split('.') }}"
+
+- name: Set Openshift Channel
+  set_fact:
+    ocp4_channel: "stable-{{ ocp4_channel[0] + '.' + ocp4_channel[1] }}"
+
+- name: Print OpenShift version
+  debug:
+    msg: "{{ ocp4_channel }}"
+
+- name: Adapt to the openshift_cluster_version LESS than 4.9
```

Comment:
this is very nice, controlling the openshift cluster version for not breaking the retrocompability with the older versions of ocp! great job @MoOyeg++